### DISCUSSION TITLE
sudo: no valid sudoers sources found, quitting (on MacBook Pro)

### DISCUSSION THREAD

**Discussion by SinuosityAI** [Homebrew/discussions#1996] [Getting started]
==> Checking for `sudo` access (which may request your password).
sudo: /etc/sudoers is world writable
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin
Need sudo access on macOS (e.g. the user admin needs to be an Administrator)!
sudo: /etc/sudoers is world writable
sudo: no valid sudoers sources found, quitting
sudo: unable to initialize policy plugin

**Comment by SMillerDev** [comment]
All of the problems are listed in the output and this isn't a brew issue. 

- You should make sure that your user is an admin
- Ensure the `/etc/sudoers` file can only be edited by admin.

**Reply by Kentzo** [reply]
*Not admin (which is likely your current user) but root:wheel. The mode must be 440 as well. Same for all the file in /etc/sudoers.d/
